src/main/java/me/jellysquid/mods/sodium/client/render/chunk/compile/executor/ChunkJobResult.java

```diff
@@ -18,7 +20,10 @@ public static <OUTPUT> ChunkJobResult<OUTPUT> successfully(OUTPUT output) {
     }
 
     public OUTPUT unwrap() {
-        if (this.throwable != null) {
+        if (this.throwable instanceof CrashException) {
```

Comment:
Should use `if (this.throwable instanceof CrashException crashException)` since it does the instanceof and type-cast in one.

Reply:
Ah, I forgot that was a feature! Been writing Java 8 for too long...